### PR TITLE
v4.025 release notes

### PR DESCRIPTION
**Pull request by lucas-hay**

### CODE REVIEW

File: site/pages/docs/versions/v4.0/v4.0-changelog-en.hbs [hunk 1]
```diff
@@ -4,12 +4,45 @@
...
+			<li>Convert pagination to a list</li>
+			<li>Wb-disable - Add a notice to disabled mode</li>
+			<li>Data-ajax: add cache busting feature</li>
+			<li>Facebook plugin</li>
+			<li>Data-ajax - Conditional loading based on HTTP referrer</li>
+			<li>Data Ajax - Working example and docs for httpref config</li>
+			<li>Add print to overlays</li>
+			<li>Footnotes: Fully disabled plugin in wbdisable mode.</li>
+			<li>Overlay Print - Only print the active dialog.</li>
+			<li>Update twitter demo and documentation<</li>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/v4.0/v4.0-changelog-en.hbs]: Got an issue here

File: site/pages/docs/versions/v4.0/v4.0.25-en.hbs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+		<li>Convert pagination to a list</li>
+		<li>Wb-disable - Add a notice to disabled mode</li>
+		<li>Data-ajax: add cache busting feature</li>
+		<li>Facebook plugin</li>
+		<li>Data-ajax - Conditional loading based on HTTP referrer</li>
+		<li>Data Ajax - Working example and docs for httpref config</li>
+		<li>Add print to overlays</li>
+		<li>Footnotes: Fully disabled plugin in wbdisable mode.</li>
+		<li>Overlay Print - Only print the active dialog.</li>
+		<li>Update twitter demo and documentation<</li>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/v4.0/v4.0.25-en.hbs]: Here too

File: site/pages/docs/versions/v4.0/v4.0.25-fr.hbs [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+		<li>Convert pagination to a list</li>
+		<li>Wb-disable - Add a notice to disabled mode</li>
+		<li>Data-ajax: add cache busting feature</li>
+		<li>Facebook plugin</li>
+		<li>Data-ajax - Conditional loading based on HTTP referrer</li>
+		<li>Data Ajax - Working example and docs for httpref config</li>
+		<li>Add print to overlays</li>
+		<li>Footnotes: Fully disabled plugin in wbdisable mode.</li>
+		<li>Overlay Print - Only print the active dialog.</li>
+		<li>Update twitter demo and documentation<</li>
```
Comment (LaurentGoderre) [inline site/pages/docs/versions/v4.0/v4.0.25-fr.hbs]: And here

**Review by LaurentGoderre — CHANGES_REQUESTED** [review]
Got some issue but it might be from the commit message. Still would be a good idea to fix

**Comment by lucas-hay** [comment]
@LaurentGoderre fixed

**Review by LaurentGoderre — APPROVED** [review]

**Pull request merged**